app/assets/javascripts/application/characters.js

```diff
@@ -0,0 +1,68 @@
+mumuki.load(() => {
+  let characters = mumuki.characters || {};
+
+  $.get('/character/animations.json',  (animations) => {
+    loadCharacters(animations.main);
+    loadAnimationGroup(animations.extra);
+  });
+
+  function loadAnimationGroup(animations) {
+    return Object.keys(animations).map((character) => {
+      newCharacter(character);
+
+      return loadAnimations(character, animations[character].clips).then(() =>
+        loadActions(character, animations[character].actions)).then(() =>
+        character);
+    });
+  }
+
+  function loadCharacters(animations) {
+    var characterFinishedLoadingPromises = loadAnimationGroup(animations);
+
+    Promise.race(characterFinishedLoadingPromises).then((character) => {
+      mumuki.presenterCharacter = characters[character];
+      placeKidsAnimations();
+    });
+  }
+
+  function newCharacter(name) {
+    characters[name] = new muvment.Character();
+  }
+
+  function loadAnimations(character, animations) {
+    return Promise.all(animations.map((svg) =>
```

Comment:
Se should move the animations loading concern to muvement in the future

Reply:
Done